app/models/vacancy.rb

```diff
@@ -38,14 +38,15 @@ class Vacancy < ApplicationRecord
   include Redis::Objects
 
   include AlgoliaSearch
+  AlgoliaSearch::IndexSettings::DEFAULT_BATCH_SIZE = 100
 
   # For guidance on sanity-checking an indexing change, read documentation/algolia_sanity_check.md
 
   # rubocop:disable Metrics/BlockLength
   # rubocop:disable Metrics/LineLength
   # There must be a better way to pass these settings to the block, but everything seems to break
   algoliasearch index_name: Rails.env.test? ? 'Vacancy_test' : 'Vacancy', auto_index: Rails.env.production?, auto_remove: Rails.env.production?, synchronous: Rails.env.test?, disable_indexing: !(Rails.env.production? || Rails.env.test?) do
-    attributes :job_roles, :job_title, :salary, :working_patterns
```

Comment:
Ah, sorry, we weren't quite on the same page here.  Leave the `job_roles` array in as we are likely to use it for presentational purposes in the client.